composer.json

```diff
@@ -11,7 +11,8 @@
         "php": "^7.1.3",
```

Comment:
Also, standard warning - PHP 7.1 is going fully EOL in 3 months, we need to be upgraded and not supporting it by then.